dist/google-keep-card.js

```diff
@@ -120,36 +115,36 @@ class GoogleKeepCard extends LitElement {
                     bottom:0;
                     left: 0;
                     right: 0;
-                    opacity: ${this._alpha};
                 }
                 #googleKeepCard .card > * {
                     position: relative;
                 }
-                
                 #googleKeepCard .noteTitle {
                     font-weight: bold;
                     margin: 0 0 14px 0;
                     font-size: 120%;
                 }
-                
                 #googleKeepCard .noteLine  {
                     margin-top: 6px;
                     margin-bottom: 6px;
                     min-height: 20px;
                 }
-                
                 #googleKeepCard hr {
                     margin: 10px 0;
-                    border: 1px solid rgba(0,0,0,0.1);
```

Comment:
That was the exact color of the separation in google keep :-) I guess we want something more visible :-) I will keep that in mind (maybe later, add an option for people to choose between two styles?)

Reply:
Yes, it was almost invisible in dark themes. I think that additional styles are not necessary, as everything can be changed with [card-mod](https://github.com/thomasloven/lovelace-card-mod)

Reply:
Yes, the google way is very subtle. But it also gets the advantage to reuse the background color of the card :-) Maybe we should just keep opacity but change the value?
Both solution (opacity change or static color) works for me since card-mod could do it both way :-)